tigerpath/majors_and_certificates/majors/EEB_2018.json

```
@@ -26,18 +26,19 @@
       "max_counted": 1,
       "min_needed": "ALL",
       "explanation": "The Department of Ecology and Evolutionary Biology requires the following prerequisites, which should normally be completed by the end of sophomore year.",
+      "completed_by_semester": 4,

```

See my comment on line 91.
```suggestion
      "completed_by_semester": 4,
      "double_counting_allowed": true,
```